client.go

```diff
@@ -210,7 +203,20 @@ func (c *Client) connect() (err error) {
 	authComplete := make(chan error)
 
 	go c.handleRawServerMessages(authComplete)
-	go c.handleOpcodes(authComplete)
+	go func() {
+		c.handleOpcodes(authComplete)
+
+		// we write to IncomingResponses only from two places:
+		// * c.handleOpcodes
+		// * c.client.SendRequest
```

Comment:
`c.client.SendRequest` never writes to the `IncomingResponses` channel. It only reads from it:

https://github.com/andreykaipov/goobs/blob/060699aab72126c89d2af14ced42023659b8d51e/api/client.go#L106-L112

Reply:
Indeed. Fixed the comment :)